<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>coolpage</class>
 <widget class="QWidget" name="coolpage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>880</width>
    <height>52</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">

QWidget#coolpage{
background-color:#333333;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1,10,1,1,1">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>Arial Black</family>
       <pointsize>18</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(255, 255, 255)</string>
     </property>
     <property name="text">
      <string>QPoolSage</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="styleSheet">
      <string notr="true">image: url(:/Photo/app.ico);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>489</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_3">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
